### PR TITLE
Feat/accessibility

### PR DESCRIPTION
**Pull request by colganmeanor**
What (if any) features are you implementing?
- no new features
What (if anything) did you refactor?
- refactored all components and JSX to include aria labels where appropriate and alt text for images. 
- removed non alcoholic option from pairing form - not enough time to build out appropriate results for this sub-feature
Were there any issues that arose?
- None
Is there anything that you need from your teammate?
- None
Any other comments, questions, or concerns?
- 100% on Lighthouse Accessibility 🔥🔥🔥

### CODE REVIEW

File: src/Components/Drink.js [hunk 1]
```diff
@@ -56,11 +56,11 @@ const Drink = () => {
...
 
     return (
-        <div className="paired-component">
+        <div aria-label="Random Drink Result" className="paired-component">
             <h3 className='drink-name'>{currentDrink.strDrink}</h3>
-            <section className='drink-info'>
-                <img className="drink-image" src={currentDrink.strDrinkThumb} />
-                <aside className='drink-specs'>
+            <section aria-label="Drink Information" className='drink-info'>
+                <img alt={`${currentDrink.strDrink} image`} className="drink-image" src={currentDrink.strDrinkThumb} />
```
Comment (raz-joan) [inline src/Components/Drink.js:62]: Nice use of the state here @colganmeanor !

File: src/Components/Favorites.js [hunk 2]
```diff
@@ -12,12 +12,12 @@ const Favorites = () => {
     const drinks = favoriteDrinks.map(drink => {
         if (favoriteDrinks) {
             return (
-                <div className='fav-drink-card' data-cy='favorite-card'>
-                    <div className='left-side-card'>
+                <div aria-label={`${drink.strDrink} Favorite Card`} className='fav-drink-card' data-cy='favorite-card'>
+                    <div aria-label= "Drink Image and Removal Button" className='left-side-card'>
                         <h3 className='fav-drink-name'>{drink.strDrink}</h3>
-                        <button className='remove-button' data-cy='remove-button' onClick={() => dispatch(removeFavoriteDrink(drink.idDrink))}>Remove</button>
+                        <button aria-label={`Remove ${drink.strDrink} from Favorite Drinks`} className='remove-button' data-cy='remove-button' onClick={() => dispatch(removeFavoriteDrink(drink.idDrink))}>Remove</button>
```
Comment (raz-joan) [inline src/Components/Favorites.js:18]: LOVE this aria label on the button. It seems very clear for users.

File: src/Components/Header.js [hunk 3]
```diff
@@ -8,9 +8,9 @@ const Header = () => {
             <NavLink to={'/'} style={{textDecoration: 'none'}}>
                 <h1 className='title' data-cy='page-title'>Board Game Bartender</h1>    
             </NavLink>
-            <div className='slogan-icon'>
+            <div aria-label="slogan and icon" className='slogan-icon'>
                 <p className='slogan'>we'll take decision making off your plate ...</p>
-                <img src={require('../assets/cocktail-glasses.png')} className='cocktail-icon'/>
+                <img alt="Stylized drawing of two cocktails in tumbler style glasses" src={require('../assets/cocktail-glasses.png')} className='cocktail-icon'/>
```
Comment (raz-joan) [inline src/Components/Header.js:13]: 😎

File: src/Components/PairingForm.js [hunk 4]
```diff
@@ -38,19 +38,19 @@ const PairingForm = () => {
     }
 
     return (
-        <div className='loading-page-bottom'>
+        <div aria-label="Game and Drink Pairing Section" className='loading-page-bottom'>
             <div className='circular-image'>
-                <img src={require('../assets/games-drinks-friends.jpeg')} />
+                <img alt="Group of friends sitting at a table playing Jenga and drinking beers" src={require('../assets/games-drinks-friends.jpeg')} />
```
Comment (raz-joan) [inline src/Components/PairingForm.js:43]: 🍻

**Review by raz-joan — APPROVED** [review]
@colganmeanor such awesome work! I really appreciate the attention to detail as you made our site accessible! 🙌🏻

File: src/Components/Drink.js (hunk 1 above)
Comment (michelecomfort) [reply src/Components/Drink.js:62]: Agreed - smart thinking!

File: src/Components/PairingForm.js [hunk 5]
```diff
@@ -38,19 +38,19 @@ const PairingForm = () => {
...
             </div>
-            <form className='game-liquor-input pairing-form' data-cy='pairing-form'>
-                <label htmlFor='game-choice' data-cy='pairing-form' className='game-input'>
-                    <input className='game-dropdown' data-cy='game-choice' id='game-choice' placeholder='Choose Your Game!' list='games' onChange={(event) => dispatch(setGameName(event.target.value))}/>
+            <form aria-label="Choose Your Game and Liquor" className='game-liquor-input pairing-form' data-cy='pairing-form' title='Game and Liquor Form'>
```
Comment (michelecomfort) [inline src/Components/PairingForm.js:45]: Looks great - well done! 😃

**Review by michelecomfort — COMMENTED** [review]
The alt labels are fantastic - I've never interpolated in state before - awesome idea!

**Pull request merged**